src/screens/SSIContactDetailsScreen/index.tsx

```diff
@@ -18,7 +18,7 @@ enum ContactTabRoutesEnum {
 }
 
 const SSIContactDetailsScreen: FC<Props> = (props: Props): JSX.Element => {
-  const { contact } = props.route.params
```

Comment:
i'm not a fan of this solution. we do not need this option here and i now need to turn it on. like discussed. all you need to do is comment out the routes in this screen. that way they do not get rendered.